Gruntfile.js

```diff
@@ -1,134 +1,149 @@
-/* eslint-env node */
+/* eslint-env node, es6 */
 /* global require */
-
 'use strict';
 
 module.exports = function (grunt) {
+  require('load-grunt-tasks')(grunt);
+  var isDev = grunt.option('env') === 'dev';
+  if (isDev) {
+    console.log('Compiling in dev mode!');
+  }
+
+  /**
+   * Style files to compile
+   * This should be a list of scss locations without the file suffix that need to be compiled
+   */
+  const styleFiles = [
```

Comment:
Another thing that I did, that I might even take a step further, is make it easier to compile multiple CSS files.

With Drupal's aggregation, D8's Libraries, and HTTP2 this might start being the best way to do things, and I wanted to make it easier to accomplish. Different packages handle adding files differently, but this array  and a little `Array.forEach` make it quite a bit easier to add things in a sane way.

Could do the same for Babel, but didn't, at the moment there's only one build tool messing with the JS, so the abstraction isn't necessary.